src/_posts/platform/app/2000-01-01-procfile.md

```diff
@@ -9,48 +9,150 @@ Procfile support is one of our
+- define a `console` process type to boot up [one-offs]({% post_url platform/app/2000-01-01-tasks %})
+  and interact with a database from the CLI.
+- ...
+
+When starting an application, Scalingo reads the Procfile and starts at least
+one Managed Runtime for each process type defined (unless it is scaled to 0).
+
+The process type also serves as a reference to manage the resources consumed by
+your application:
+- it allows to scale the Managed Runtime(s) of the specified process type.
+- it allows to use different plans for the Managed Runtimes, depending on the
+  workload they have to handle.
+
+Example:
+
+Let's say your application has a web frontend that handles HTTP requests and a
+worker that does some heavy computing. They both share the same code base, but
+they don't start the same way and they don't have the same computing power
+requirements. In such a case, you'd have to instruct the platform to start two
+process types, which must be defined in your project's Procfile:
+- one called `web`, to instruct the platform how to start your web server (the
+  frontend).
+- another one, called `worker`, to instruct the platform how to start your...
+  worker.
+
+The `web` process type could be set to use an `L` plan (which means the
+platform will run the corresponding command in an `L` Managed Runtime), whereas
+the `worker` process type could be set to use a `2XL` plan.
+
+Now, let's say that your single `worker` Managed Runtime is a bit under
+pressure. Using the process type, you can instruct the platform to boot up a
+few more Managed Runtimes to handle the load (in the command below, we tell the
+platform that we want three `2XL` Managed Runtimes for the `worker` process
+type):
+
+```bash
+scalingo --app my-app scale worker:3:2XL
 ```
 
-By default, when you first try to deploy your application, we'll try to
-launch the `web` process. We try to determine it without your help, but in
-some cases it is required to define it, as in the previous example. If your
-application requires a `Procfile` and you haven't defined it, your deployment
-will fail and a log entry will attest the lack of Procfile.
+### Special Process Types
+
+By default, Scalingo uses three process types: `web`, `tcp` and `postdeploy`.
+**These process types have specific use cases** and the platform will only use
+them for what they are designed.
+
+#### The `web` Process Type
+
+The `web` process type defines how to start and run the Managed Runtime
+handling HTTP requests.
+
+It's the only process type that can receive external HTTP requests from
+Scalingo's routers. Consequently, if your application has a web server, it MUST
+have a `web` process type binded on `$PORT` to be publicly reachable.
+
+The `web` process type is often the only one required.
+
+{% note %}
+Almost all our buildpacks define a default `web` process type so you don't have
+to care about it. If you don't need it, you can scale it to 0 to avoid unwanted
+costs.
+{% endnote %}
+
```

Comment:
I think it could be nice to make a sentence with web-less app and make a link to the page: https://doc.scalingo.com/platform/app/web-less-app#deploy-a-web-less-application

Reply:
Also done in https://github.com/Scalingo/documentation/pull/2020/commits/64da5f624d74d705bb2f074587d77ad065c7211c

Thanks for your feedback and inputs @brandon-welsch <3

Reply:
Thanks to you, it's a niceeeee improvement of this doc page 🙏🏻 🚀